### PR TITLE
Bugfix for proxying unavailable file in local file system

### PR DESCRIPTION
**Pull request by bartoszglow**
## Description

Check for existance of file in local file system to be proxied. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: lib/proxy.js [hunk 1]
```diff
@@ -50,8 +50,10 @@ const proxy = function(CONFIG) {
     function (req, res) {
       for (let [key, value] of routeMap) {
         const urlMatch = req.url.match(key);
-        if (urlMatch) {
-          fileSystem.createReadStream(req.url.replace(urlMatch[0], value).split('?')[0]).pipe(res);
+        const filePath = urlMatch && req.url.replace(urlMatch[0], value).split('?')[0];
+
+        if (urlMatch && fileSystem.existsSync(filePath)) {
```
Comment (vtsaplin) [inline lib/proxy.js:55]: Should it be `if (filePath && fileSystem.existsSync(filePath)) {`?

Comment (bartoszglow) [reply lib/proxy.js:55]: Why would we need to check for filePath? If it is `undefined` or `''` then fileSystem.existsSync will still return false.

Comment (vtsaplin) [reply lib/proxy.js:55]: Well then the check should not include `urlMatch`. Something like this?
```
if (fileSystem.existsSync(filePath)) {
```

Comment (bartoszglow) [reply lib/proxy.js:55]: correct, `urlMatch` is not required anymore

**Review by vtsaplin — APPROVED** [review]

**Review by vladbailescu — APPROVED** [review]

**Pull request merged**